src/helpers/profilePictureService.ts

```diff
@@ -1,6 +1,38 @@
 import { PrismaClient } from ".prisma/client";
 import { getEmployeeById } from "./employeeService";
 const prisma = new PrismaClient();
+import dotenv from "dotenv";
+dotenv.config();
+const cloudinary = require("cloudinary").v2;
+
+export const uploadImageToCloudinary = async (createReadStream) => {
+	cloudinary.config({
+		cloud_name: process.env.CLOUDINARY_CLOUD_NAME,
+		api_key: process.env.CLOUDINARY_API_KEY,
+		api_secret: process.env.CLOUDINARY_API_SECRET,
+	});
+
+	const { secure_url: profilePictureUrl } = await new Promise(
```

Comment:
This code seems a bit complex for what it's trying to do. Is this copied from somewhere?

Reply:
Yes. I had checked one post. It seemed simpler.

Reply:
Changes have been pushed in new commit